test2/faces40/cdi/src/main/java/com/sun/faces/test/javaee8/cdi/ConfigurationBean.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021 Contributors to Eclipse Foundation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package com.sun.faces.test.javaee8.cdi;
+
+import static jakarta.faces.annotation.FacesConfig.Version.JSF_2_3;
+
+import jakarta.faces.annotation.FacesConfig;
+
+@FacesConfig(
```

Comment:
Is this still necessary?

Reply:
It's the alternative to map the Facelets servlet automatically, not to set the version to 2.3 or anything. We do need to define what the version now means in another issue.